### PR TITLE
Poll on session roomId on login

### LINKED ISSUE #56: Check user belongs to an active session when logging in

### PR DESCRIPTION
**Pull request by rachtan27**
closes #56

### CODE REVIEW

File: src/collaboration-service/controllers/session-controller.js [hunk 1]
```diff
@@ -118,3 +118,18 @@ exports.editSession = async (req, res) => {
...
+    const session = await Session.findOne({
+      $or: [
+        { firstUserId: uid },
+        { secondUserId: uid }
+      ]
+    });
+    return res.status(200).json(session);
+  } catch (err) {
+    console.log(err);
+    return res.status(500).json({ message: "Error fetching session" });
```
Comment (changjy44) [inline src/collaboration-service/controllers/session-controller.js:133]: I think returning an error when the session does not exist is not good? There might be other cases to handle.

Maybe can use aggregate instead?

**Review by changjy44 — CHANGES_REQUESTED** [review]
Use aggregate or other functions instead of getting an error

File: src/collaboration-service/controllers/session-controller.js (hunk 1 above)
Comment (ish1506) [reply src/collaboration-service/controllers/session-controller.js:133]: The Mongoose findOne returns null if no session found, so this is fine if the FE handles the null case

Comment (rachtan27) [reply src/collaboration-service/controllers/session-controller.js:133]: > I think returning an error when the session does not exist is not good? There might be other cases to handle.
> 
> Maybe can use aggregate instead?

When no session, theres no error, jus that session is null.
so on front end it checks "if session" to handle null case.

Comment (changjy44) [reply src/collaboration-service/controllers/session-controller.js:133]: Should be okay then

**Review by ish1506 — APPROVED** [review]
LGTM

**Pull request merged**